sdccc/src/main/java/com/draeger/medical/sdccc/configuration/TestParameterConfig.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * This Source Code Form is subject to the terms of the MIT License.
+ * Copyright (c) 2024 Draegerwerk AG & Co. KGaA.
+ *
+ * SPDX-License-Identifier: MIT
+ */
+package com.draeger.medical.sdccc.configuration
+
+/**
+ * Test parameter configuration for SDCcc.
+ *
+ * @see DefaultTestParameterConfig
+ */
+object TestParameterConfig {
+    /*
+     * Test parameter configuration
+     */
+    private const val TEST_PARAMETER = "TestParameter."
```

Comment:
Probably shouldn't be private

Reply:
I would leave it private, otherwise the Toml parser will interpret this as configuration key for a test parameter.